source/includes/fact-authenticate-with-java-gssapi.rst

```diff
@@ -0,0 +1,48 @@
+GSSAPI (Kerberos) Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important::
+
+   The Java driver maintainers have only confirmed the content of this
+   document on Linux using Java 6 and Java 7, on OS X using Java 7,
+   and on Windows with SSPI using Java 7.
+
+   These features are only present in `MongoDB Enterprise
+   <https://www.mongodb.com/products/mongodb-enterprise>`_.
+
+To authenticate to a MongoDB cluster using Kerberos, you must supply
+the Kerberos user name and specify the Kerberos authentication
+mechanism:
+
+.. code-block:: java
+
+   MongoCredential credential = MongoCredential.createGSSAPICredential("user1@MYREALM.ME");
+
+   MongoClient mongoClient = new MongoClient(new ServerAddress(server), Arrays.asList(credential));
+
+With Kerberos you specify neither the password nor the database name,
+as authentication is performed via the Kerberos key distribution
+center (KDC).
+
+To actually run a program that authenticates with Kerberos, you will
+typically need to specify several system properties so that the
+underlying GSSAPI Java libraries can acquire a Kerberos ticket:
+
+.. code-block:: properties
+
+   java.security.krb5.realm=MYREALM.ME
+   java.security.krb5.kdc=mykdc.myrealm.me
+
+.. note::
+
+   Many authentication problems with GSSAPI/Kerberos occur due to
```

Comment:
Yes, aside from some minor copyediting. The note block is the salient bit
for the ticket.

On Tue, Mar 17, 2015 at 10:39 AM, Jeff Yemin <notifications@github.com>
wrote:

> In source/includes/fact-authenticate-with-java-gssapi.rst
> <https://github.com/mongodb/docs-ecosystem/pull/262#discussion_r26577912>:
>
> > +With Kerberos you specify neither the password nor the database name,
> > +as authentication is performed via the Kerberos key distribution
> > +center (KDC).
> > +
> > +To actually run a program that authenticates with Kerberos, you will
> > +typically need to specify several system properties so that the
> > +underlying GSSAPI Java libraries can acquire a Kerberos ticket:
> > +
> > +.. code-block:: properties
> > +
> > +   java.security.krb5.realm=MYREALM.ME
> > +   java.security.krb5.kdc=mykdc.myrealm.me
> > +
> > +.. note::
> > +
> > +   Many authentication problems with GSSAPI/Kerberos occur due to
>
> So is this the only bit that has really changed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mongodb/docs-ecosystem/pull/262/files#r26577912>.
>



-- 
{
  name     : *"*Michael Paik*", *
  title    : "Director of Documentation",
  company  : "MongoDB, Inc.",
  web      : ["www.mongodb.org", "www.mongodb.com"]
}